src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -256,6 +256,81 @@ ledgerFromRequest(T& ledger, JsonContext& context)
 
     return Status::OK;
 }
+
+template <class T>
+Status
+ledgersFromRequest(std::vector<T>& ledgers, JsonContext& context)
```

Comment:
Instead of writing a new function, why can't you just call `ledgerFromRequest` in a loop? There is a lot of code duplication between `ledgersFromRequest` and `ledgerFromRequest`